modules/administration-guide/pages/configuring-the-open-vsx-registry-url.adoc

```diff
@@ -0,0 +1,28 @@
+:_content-type: PROCEDURE
+:description: Configuring the Open VSX registry URL for all {prod} workspaces
+:keywords: administration guide, configuring, openvsx, registry
+:navtitle: Open VSX registry URL
+:page-aliases:
+
+[id="configuring-the-open-vsx-registry-url"]
+= Configuring the Open VSX registry URL
+
+To search and install extensions, the Visual Studio Code editor is using an embedded Open VSX registry instance.
+You can configure {prod-short} to rather use your Open VSX registry instance.
```

Comment:
Maybe:

You can configure {prod-short} to use your Open VSX registry instance instead.

Reply:
I believe style guide doesn't like "instead".

Reply:
ok, maybe something like:

"You can also configure {prod-short} to use your own Open VSX registry instance" and add on "rather than the embedded one" if required.

It is just the position of the word "rather " that feels off.

Reply:
```suggestion
You can also configure {prod-short} to use your own Open VSX registry instance rather than the embedded one.
```